major_event_log/models.py

```diff
@@ -30,6 +30,12 @@ class Event(models.Model):
                                     "Name will appear as the Reporting Agent")
     contact_email = models.EmailField()
 
+    def is_success(self):
+        if self.get_outcome_display() == "Success":
```

Comment:
It also makes contains those "settings" to a single location. If, for whatever reason, we need to changes, we won't have to dig through the code to make it happen.